sympy/utilities/tests/test_lambdify.py

```diff
@@ -9,6 +9,14 @@
 from sympy.utilities.pytest import skip
 from sympy.utilities.decorator import conserve_mpmath_dps
 from sympy.external import import_module
+# for new test cases of scipy
+from sympy.functions.special.bessel import *
+from sympy.functions.special.hyper import * 
+from sympy.functions.special.hyper import *
+from sympy.functions.special.gamma_functions import *
+from sympy.functions.special.gamma_functions import *
```

Comment:
Please don't use the `from x import *` style of import. Always make it explicit.